policyengine_us/parameters/gov/states/oh/tax/income/agi_threshold.yaml

```diff
@@ -1,9 +1,15 @@
 description: Ohio requires filers with this adjusted gross income or more to pay individual income tax.
 values:
   2021-01-01: 25_000
+  2022-01-01: 26_050
+  2023-01-01: 26_050
```

Comment:
```suggestion
```

Reply:
dont need to duplicate the value if it says the same - just the reference is enough